src/vm/context/run_context.rs

```diff
@@ -1,24 +1,33 @@
 use crate::bigint;
 use crate::types::instruction::{Instruction, Op1Addr, Register};
-use crate::types::relocatable::MaybeRelocatable;
+use crate::types::relocatable::{MaybeRelocatable, Relocatable};
 use crate::vm::errors::vm_errors::VirtualMachineError;
 use num_bigint::BigInt;
 
 pub struct RunContext {
-    pub pc: MaybeRelocatable,
-    pub ap: MaybeRelocatable,
-    pub fp: MaybeRelocatable,
+    pub pc: Relocatable,
+    pub ap: usize,
+    pub fp: usize,
     pub prime: BigInt,
 }
 
 impl RunContext {
+    pub fn get_pc(&self) -> MaybeRelocatable {
```

Comment:
In the future, this method should return a Relocatable
https://github.com/lambdaclass/cairo-rs/issues/391